frontend/controller/actions/group.js

```diff
@@ -524,6 +527,7 @@ export default (sbp('sbp/selectors/register', {
       console.error('gi.actions/group/join failed!', e)
       throw new GIErrorUIRuntimeError(L('Failed to join the group: {codeError}', { codeError: e.message }))
     } finally {
+      sbp('okTurtles.data/set', 'JOINING_GROUP-' + params.contractID, false)
```

Comment:
This function returning does _not_ mean that the joining process is complete. It's complete when keys are received.